<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(243, 220, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="2" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Bodoni MT&quot;;
background-color: rgb(172, 169, 255);
border-radius: 10px;
border-bottom-style:solid;
border-bottom-width: 5px;
border-bottom-color: rgb(109, 127, 209);
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;NOTES&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="listofgroups">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="add_group_button">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Bodoni MT&quot;;
background-color: rgb(172, 169, 255);
border-radius: 10px;
border-bottom-style:solid;
border-bottom-width: 5px;
border-bottom-color: rgb(109, 127, 209);</string>
        </property>
        <property name="text">
         <string>Add group of notes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="archive_button">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Bodoni MT&quot;;
background-color: rgb(172, 169, 255);
border-radius: 10px;
border-bottom-style:solid;
border-bottom-width: 5px;
border-bottom-color: rgb(109, 127, 209);
</string>
        </property>
        <property name="text">
         <string>Archive</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delete_button">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Bodoni MT&quot;;
background-color: rgb(172, 169, 255);
border-radius: 10px;
border-bottom-style:solid;
border-bottom-width: 5px;
border-bottom-color: rgb(109, 127, 209);
</string>
        </property>
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="exit_button">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Bodoni MT&quot;;
background-color: rgb(172, 169, 255);
border-radius: 10px;
border-bottom-style:solid;
border-bottom-width: 5px;
border-bottom-color: rgb(109, 127, 209);
</string>
        </property>
        <property name="text">
         <string>Save and Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listofnotes"/>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="styleSheet">
         <string notr="true">font: 11pt &quot;Bodoni MT&quot;;
background-color: rgb(172, 169, 255);
border-radius: 0px;
border-bottom-style:solid;
border-bottom-width: 5px;
border-bottom-color: rgb(109, 127, 209);</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
